protos/sources/vega/commands/v1/commands.proto

```diff
@@ -132,6 +160,20 @@ message LiquidityProvisionAmendment {
   string reference = 6; // empty string means no change
 }
 
+// Amend a liquidity provision request
+message SpotLiquidityProvisionAmendment {
+  // Unique ID for the market with the liquidity provision to be amended.
+  string market_id = 1;
+
+  // From here at least one of the following is required to consider the command valid.
+  string buy_commitment_amount = 2; // empty strings means no change
```

Comment:
We should just make these `optional` fields give they are optional, and then we don't have to rely on an empty string having a special meaning.